ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/config/DiscoveryConfiguration.java

```diff
@@ -148,6 +148,19 @@ public DiscoveryConfiguration setBootstrapPeers(final Collection<?> bootstrapPee
     return this;
   }
 
+  public static List<Peer> getBootstrapPeersFromUnknownCollection(
```

Comment:
Maybe change `getBootstrapPeersFromUnknownCollection` for `getBootstrapPeersFromGenericCollection`?